pkg/convert/byteConverter.go

```diff
@@ -0,0 +1,32 @@
+package convert
+
+import (
+	"encoding/binary"
+	"unicode/utf16"
+)
+
+// Encode uint64 to byte array.
+func Uint64ToByteArrayU8(u64 uint64) []byte {
+	encode := 8
+	b := make([]byte, encode)
+	binary.LittleEndian.PutUint64(b, u64)
+
+	return b
+}
+
+// Decode uint32 to string.
+func EncodeUint32ToUTF8String(numberToEncode uint32) string {
```

Comment:
Go is a typed language, it's useless to set type name in the name of variables
```suggestion
func u32ToString(number uint32) string {
```